public/docs/_examples/i18n/e2e-spec.ts

```diff
@@ -0,0 +1,13 @@
+/// <reference path='../_protractor/e2e.d.ts' />
+'use strict';
```

Comment:
In our case yes, because we there's a couple of es2015 things that node requires `'use strict';` to support.